### PR TITLE
Create VideoGame_Beginning

### PR DESCRIPTION
**Pull request by Fintech226**
Create Specialization

### CODE REVIEW

File: VideoGame_Beginning [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Resaons for Argument
```
Comment (Fintech226) [inline VideoGame_Beginning:1]: Gaming trends and how predicted in the future. AI and critic scores

File: VideoGame_Beginning [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+Dataset
```
Comment (Fintech226) [inline VideoGame_Beginning:3]: Video Game Sales Kaggle Dataset

File: VideoGame_Beginning [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+Questions
```
Comment (Fintech226) [inline VideoGame_Beginning:5]: Where have Gaming sales come? Global Players to User Count. What is the role of Critic Scores? Can they get better over time through successful game sales?

File: VideoGame_Beginning [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+Theory
```
Comment (Fintech226) [inline VideoGame_Beginning:7]: Critic Scores predict Global players better as time goes on. User Scores predict more participation in gaming as time goes on.

File: VideoGame_Beginning [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+Explanation
```
Comment (Fintech226) [inline VideoGame_Beginning:9]: Video games have increased total global players, however have not retained a total global player increase for titles as time goes on. Only a increase in user scores have increased consistently over time where total global players for a title do not increase consistently by year.

File: VideoGame_Beginning [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+Thesis
```
Comment (Fintech226) [inline VideoGame_Beginning:11]: How does critic score relate to global players?

How does User Score relate to/increase US game sales?

Which video game maker benefits the most by critic and user scores?

File: VideoGame_Beginning [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+
+Data
```
Comment (Fintech226) [inline VideoGame_Beginning:13]: | Name | Platform | Year_of_Release | Genre | Publisher | NA_players | EU_players | JP_players | Other_players | Global_players | Critic_Score | Critic_Count | User_Score | User_Count | Developer | Rating

File: VideoGame_Beginning (hunk 2 above)
Comment (Fintech226) [reply VideoGame_Beginning:3]: https://www.kaggle.com/datasets/gregorut/videogamesales

File: VideoGame_Beginning [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+
+Conclusions of Data
```
Comment (Fintech226) [inline VideoGame_Beginning:15]: Descriptive Statistics
Regression
Specific exploration ways to solve questions

File: VideoGame_Beginning [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+
+Conclusion
```
Comment (Fintech226) [inline VideoGame_Beginning:17]: What do Critic Score and User Score say about global players? 
Answers to questions.
Implications to answers.

File: VideoGame_Beginning (hunk 1 above)
Comment (rwhiting1979) [reply VideoGame_Beginning:1]: Looks good!

**Review by rwhiting1979 — APPROVED** [review]
Great outline for the project!

File: VideoGame_Beginning (hunk 8 above)
Comment (Fintech226) [reply VideoGame_Beginning:15]: Coefficients:
               Estimate Std. Error t value Pr(>|t|)    
(Intercept)  -0.7149775  0.0486996  -14.68   <2e-16 ***
Critic_Score  0.0154849  0.0006921   22.37   <2e-16


Coefficients:
             Estimate Std. Error t value Pr(>|t|)    
(Intercept) 3.099e-01  1.072e-02   28.90   <2e-16 ***
User_Count  4.065e-04  1.835e-05   22.15   <2e-16 ***

     ...1           Name             Platform         Year_of_Release   
 Min.   :    1   Length:16719       Length:16719       Length:16719      
 1st Qu.: 4180   Class :character   Class :character   Class :character  
 Median : 8360   Mode  :character   Mode  :character   Mode  :character  
 Mean   : 8360                                                           
 3rd Qu.:12540                                                           
 Max.   :16719                                                           
                                                                         
    Genre            Publisher           NA_players        EU_players    
 Length:16719       Length:16719       Min.   : 0.0000   Min.   : 0.000  
 Class :character   Class :character   1st Qu.: 0.0000   1st Qu.: 0.000  
 Mode  :character   Mode  :character   Median : 0.0800   Median : 0.020  
                                       Mean   : 0.2633   Mean   : 0.145  
                                       3rd Qu.: 0.2400   3rd Qu.: 0.110  
                                       Max.   :41.3600   Max.   :28.960  
                                                                         
   JP_players      Other_players      Global_players     Critic_Score  
 Min.   : 0.0000   Min.   : 0.00000   Min.   : 0.0100   Min.   :13.00  
 1st Qu.: 0.0000   1st Qu.: 0.00000   1st Qu.: 0.0600   1st Qu.:60.00  
 Median : 0.0000   Median : 0.01000   Median : 0.1700   Median :71.00  
 Mean   : 0.0776   Mean   : 0.04733   Mean   : 0.5335   Mean   :68.97  
 3rd Qu.: 0.0400   3rd Qu.: 0.03000   3rd Qu.: 0.4700   3rd Qu.:79.00  
 Max.   :10.2200   Max.   :10.57000   Max.   :82.5300   Max.   :98.00  
                                                        NA's   :8582   
  Critic_Count     User_Score          User_Count       Developer        
 Min.   :  3.00   Length:16719       Min.   :    4.0   Length:16719      
 1st Qu.: 12.00   Class :character   1st Qu.:   10.0   Class :character  
 Median : 21.00   Mode  :character   Median :   24.0   Mode  :character  
 Mean   : 26.36                      Mean   :  162.2                     
 3rd Qu.: 36.00                      3rd Qu.:   81.0                     
 Max.   :113.00                      Max.   :10665.0                     
 NA's   :8582                        NA's   :9129                        
    Rating         
 Length:16719      
 Class :character

Comment (Fintech226) [reply VideoGame_Beginning:15]: (SQL to Video Game Manufacturer winner or most influential)

File: VideoGame_Beginning (hunk 9 above)
Comment (Fintech226) [reply VideoGame_Beginning:17]: The higher the Critic score and the higher the user score the more north American players buy the game. Critic Scores are more influential per north american purchases. Nintendo does the best with North American player count.

**Comment by Fintech226** [comment]
import pandas as pd

df=pd.read_csv(r'C:\Users\pwals\OneDrive\Desktop\video_game.csv')
print(df)

df['NA_players'].describe()

df['Critic_Score'].describe()

df['User_Count'].describe()

df['User_Count'].describe()

from sklearn import datasets
data=df

target = pd.DataFrame(data.target, columns=["MEDV"])
X = df["Critic_Score"]
y = df["NA_players"]
model = sm.OLS(y, X).fit()
predictions = model.predict(X) 
model.summary()

**Comment by Fintech226** [comment]
image

**Comment by Fintech226** [comment]
image

**Comment by rwhiting1979** [comment]
The graphs look awesome

**Pull request merged**